rules/windows/powershell/powershell_script/posh_ps_x509enrollment.yml

```diff
@@ -0,0 +1,25 @@
+title: Suspicious X509Enrollment - Ps Script
+id: 504d63cb-0dba-4d02-8531-e72981aace2c
+related:
+    - id: 114de787-4eb2-48cc-abdb-c0b449f93ea4
+      type: similar
+status: experimental
+description: Detect use of X509Enrollment
+references:
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse?slide=42
+    - https://learn.microsoft.com/en-us/dotnet/api/microsoft.hpc.scheduler.store.cx509enrollmentwebclassfactoryclass?view=hpc-sdk-5.1.6115
+author: frack113
+date: 2022/12/23
+logsource:
+    product: windows
+    category: ps_script
+    definition: Script block logging must be enabled
+detection:
+    selection:
+        ScriptBlockText|contains:
+            - 'X509Enrollment.CBinaryConverter'
+            - '884e2002-217d-11da-b2a4-000e7bbb2b09'
```

Comment:
Can't find malicious scripts on VT using this and the screenshot didn't show any result for this. Can you find examples?

Reply:
https://hybrid-analysis.com/sample/3db05b03b7384ec90a16fdb0a23292aa7aa1090d795802c58a2beb844fc5d763/6341f9c40d08c7439445ff13
Not sure can be set to low

Reply:
Here it is https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse?slide=41

We can keep it as a medium rule and include this in the reference